portfolio/src/components/Connect.vue

```diff
@@ -1,15 +1,116 @@
 <template>
     <div id="connect-container">
-        <span>Coming in week 3!</span>
+        <h1 style="width: 100%; margin-top: 1rem;">Connect</h1>
+        <div id="social-media-container">            
+            <img class="media-box" src="/assets/GitHub.svg" alt="linkedin">
+            <img class="media-box" src="/assets/GitHub.svg" alt="linkedin">
+            <img class="media-box" src="/assets/GitHub.svg" alt="linkedin">
+        </div>
+
+        <h1 style="width: 100%;">Leave a Comment</h1>
+        <div id="comments-container">
+            <div id="message-input-container" class="comment">
+                <v-text-field 
+                    style="width: 100%" 
+                    label="Name"
+                    v-model="name"
+                    color="#DBDBDB"
```

Comment:
this seems like a value used many times. Are you able to create a constant for this value with readable name e.g. main_text_color or etc and reference it here?

Reply:
Ok, made a constants file that exports a COLORS. Will be in next commit